src/VisualStudio/Core/Impl/Options/Style/NamingPreferences/NamingStyleOptionPageViewModel.cs

```diff
@@ -275,6 +275,34 @@ public bool CanMoveDown
                 }
             }
 
+            public string MoveUpAutomationText
```

Comment:
Around line 121 in the XAML. @dpoeschl was this a case where we were consuming a property that wasn't defined?